be/src/runtime/tablets_channel.cpp

```diff
@@ -25,11 +25,9 @@
 
 namespace doris {
 
-TabletsChannel::TabletsChannel(
-        const TabletsChannelKey& key,
-        MemTracker* mem_tracker): 
-    _key(key), _closed_senders(64) {
-    _mem_tracker.reset(new MemTracker(-1, "tablets channel", mem_tracker));
+TabletsChannel::TabletsChannel(const TabletsChannelKey& key, MemTracker* parent) :
```

Comment:
I don't think parent is a good name, parent looks like parent of tabletsChannel not only a MemTracker.